docs/getting-started/what-is-dotnet.md

```diff
@@ -73,19 +73,19 @@ CoreCLR, or the common language runtime found in .NET Core, is a lightweight run
 
 - A just-in-time (JIT) compiler. which compiles IL, or .NET intermediate language, to native machine code at runtime. On some architectures, the JIT compiler includes support for SIMD hardware acceleration.
 
-- An exception handling mechanism. The runtime, as well as library code, throws exceptions, which you can handle with try/catch statements.   
+- An exception handling mechanism, which allows you to handle exceptions with try/catch statements.   
 
 ##The Class Library. 
 
-The class library available in .NET Core is very much like the .NET Framework Class Library for the Windows desktop, with one major difference. Whereas on the Windows desktop is a monolithic library that is part of the operating system and is maintained by Windows Update, on .NET Core it is a modular and factored collection of individual libraries organized by functionality. 
+The class library available in .NET Core is very much like the .NET Framework Class Library for the Windows desktop, with one major difference. On the Windows desktop, it is a monolithic library that is part of the operating system and is maintained by Windows Update. On .NET Core it is a modular and factored collection of individual libraries organized by functionality. 
```

Comment:
78: Stray period.
80: "The major difference between the .NET Core class library and the .NET Framework Class Library is modularity. The .NET Framework Class Library is a monolithic library that is part of the operating system and is maintained by Windows Update. The .NET Core equivalent is a modular and factored collection of individual libraries."